cookbooks/fb_grub/attributes/default.rb

```diff
@@ -65,6 +70,7 @@
   'timeout' => 5,
   'use_labels' => false,
   'version' => version,
+  'force_write_both_efi_and_bios' => true,
```

Comment:
This is referenced as `force_both_efi_and_bios` later on, so one of them is wrong. Also, you should add it to the README.